course_catalog/etl/podcast.py

```diff
@@ -199,3 +200,73 @@ def transform(extracted_podcasts):
         except AttributeError:
             log.exception("Error parsing podcast data from %s", config_data["rss_url"])
             continue
+
+
+def get_all_mit_podcasts_channel_rss():
+    """
+    Get channel information for the MIT aggregate podcast
+    Returns:
+        Beautiful soup object of the rss for the  MIT aggregate podcast, excluding episodes
+    """
+    current_timestamp = datetime.now().strftime(TIMESTAMP_FORMAT)
+
+    rss = """<?xml version='1.0' encoding='UTF-8'?>
+    <rss xmlns:itunes="http://www.itunes.com/dtds/podcast-1.0.dtd" version="2.0">
+        <channel>
+            <title>MIT Open Aggregated Podcast Feed</title>
+            <link>https://open.mit.edu/podcasts/<link>
+            <language>en-us</language>
+            <pubDate> {0} </pubDate>
+            <lastBuildDate> {0} </lastBuildDate>
+            <ttl>60</ttl>
+            <itunes:subtitle>Episodes from podcasts from around MIT</itunes:subtitle>
+            <itunes:author>MIT Open Learning</itunes:author>
+            <itunes:summary>Episodes from podcasts from around MIT</itunes:summary>
+            <description>Episodes from podcasts from around MIT</description>
+            <itunes:owner>
+                <itunes:name>MIT Open Learning</itunes:name>
+                <itunes:email>odl-discussions-support@mit.edu</itunes:email>
+            </itunes:owner>
+            <itunes:explicit>no</itunes:explicit>
+            <itunes:category text="Education"/></itunes:category>
+        </channel>
+    </rss>""".format(
+        current_timestamp
+    )
+    return bs(rss, "xml")
+
+
+def generate_rss(episode_rss_list):
+    """
+    Creates and saves an rss file for the MIT aggregate podcast
+
+    Args:
+        episode_rss_list (list of Beautiful soup objects): the rss data for podcast episodes
+
+    """
+
+    channel_rss = get_all_mit_podcasts_channel_rss()
+    episode_rss_list = sorted(
+        episode_rss_list,
+        key=lambda episode: datetime.strptime(episode.pubDate.text, TIMESTAMP_FORMAT),
+        reverse=True,
+    )
+
+    for episode in episode_rss_list:
+        channel_rss.channel.append(episode)
+
+    with open(RSS_FILE_LOCATION, "w") as file:
```

Comment:
I don't think we should be writing to our own file system. For one, anything written will be erased on the next restart of the heroku server. Instead, we could store the relevant parts of the RSS in the database in a `PodcastEpisode` field or a field on a related model, then generate the XML from that when the user views `podcast_rss_feed`.